composer.json

```diff
@@ -14,7 +14,8 @@
     ],
 
     "require": {
-        "bolt/filesystem": "^2.0",
+        "bolt/filesystem": "^2.1",
+        "contao/imagine-svg": "^0.1.2",
```

Comment:
Ah, just that while semver says that `Y`s are majors when `X` is a zero, nearly nobody actually deos that, hence one of the reasons Composer switched their default constraint marker to strict.